src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -151,20 +170,25 @@ else if (m_climberState == ClimberState.RETRACTING_HOOKS) {
             // Once one side goes high, we need to stop that winch until the other hook "catches up"
             // Note that when we enetered this state, both winches started retracting. We only need to stop them one at a time until
             // the robot is level again and then we will go to the CLIMBING state
-            if (m_rollAngle > ROLL_ANGLE_TOLERANCE) {
+
+            // TODO: what about if they both engage at the same time and stay ~level?
```

Comment:
Can we capture this TODO in a more visible place, and decide if this needs work?

Reply:
fixed comment